src/Modules/Workspaces/Commands.Workspaces/help/Remove-PowerBIWorkspaceUser.md

```diff
@@ -0,0 +1,95 @@
+---
+external help file: Microsoft.PowerBI.Commands.Workspaces.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Workspaces
+online version:
+schema: 2.0.0
+---
+
+# Remove-PowerBIWorkspaceUser
+
+## SYNOPSIS
+Removes permissions for a given user to PowerBI Workspace
+
+## SYNTAX
+
+```
+Remove-PowerBIWorkspaceUser [-Scope <PowerBIUserScope>] -Id <Guid> -UserEmailAddress <String>
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Remove-PowerBIWorkspaceUser -Scope Organization -Id 23FCBDBD-A979-45D8-B1C8-6D21E0F4BE50 -UserEmailAddress john@contoso.com
+```
+
+### Example 2
+```powershell
+PS C:\> Remove-PowerBIWorkspaceUser -Scope Individual -Id 23FCBDBD-A979-45D8-B1C8-6D21E0F4BE50 -UserEmailAddress john@contoso.com
+```
+
+## PARAMETERS
+
+### -Id
+Group or Workspace Id for which user has to be removed
+
+```yaml
+Type: Guid
+Parameter Sets: (All)
+Aliases: GroupId, WorkspaceId
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Scope
+Individual or Organization
```

Comment:
Mention Default is Individual. We should also say what those scopes do

Reply:
Resolved. Updated description for Scope in both cmdlets.